OOP_HW1/src/main/java/Order.java

```diff
@@ -0,0 +1,28 @@
+public class Order {
+    private TicketType type;
+    private int quantity;
+
+    public Order() {
+    }
+
+    public Order(TicketType t, int q) {
+        type = t;
+        quantity = q;
+    }
+
+    public void setTicketType(TicketType t) {
+        type = t;
+    }
+
+    public void setQuantity(int q) {
+        quantity = q;
+    }
+
+    public TicketType getTicketType() {
```

Comment:
In Java's name convension, we use camel case on function name.
Here's the suggestion for you.

```suggestion
    public TicketType GetTicketType() {
```